subprojects/docs/src/docs/release/notes.md

```diff
@@ -79,6 +79,30 @@ The [PMD](userguide/pmd_plugin.html) and [CodeNarc](userguide/pmd_plugin.html) p
 
 In Java projects, these tools will use the same version of Java required by the project. In other types of projects, they will use the same version of Java that is used by the Gradle daemon.
 
+#### Improvements for `buildSrc` builds
+This release includes several improvements for [`buildSrc`](userguide/organizing_gradle_projects#sec:build_sources) builds to make them behave similarly to an [included build](userguide/composite_builds#composite_build_intro).
+
+##### Run `buildSrc` tasks directly
+It is now possible to run the tasks of `buildSrc` from the command-line, using the same syntax used for the tasks of included builds.
+For example, you can use `gradle buildSrc:build` to run the `build` task in the `buildSrc` build.
+
+TODO - link to running included build tasks
+
+#### `buildSrc` can include other builds
+The `buildSrc` build can now include other builds by declaring them in `buildSrc/settings.gradle.kts` or `buildSrc/settings.gradle`.
+You can use `pluginsManagement { includeBuild(someDir) }` or `includeBuild(someDir)` in this settings script to make other builds available for `buildSrc`
+
+TODO - link to declaring included builds
+
+#### Tests for `buildSrc` are no longer automatically run
+When Gradle builds the output of `buildSrc` it only runs the tasks that produce that output. It no longer runs the `build` task.
+In particular, this means that the tests of `buildSrc` and its subprojects are not built and executed when they are not needed.
+
+TODO - you can run these tasks from the command-line or edit buildSrc to restore the old behaviour; link to upgrade guide 
+
+#### Init scripts are applied to `buildSrc`
```

Comment:
```suggestion
##### `buildSrc` can include other builds
The `buildSrc` build can now include other builds by declaring them in `buildSrc/settings.gradle.kts` or `buildSrc/settings.gradle`.
You can use `pluginsManagement { includeBuild(someDir) }` or `includeBuild(someDir)` in this settings script to make other builds available for `buildSrc`

TODO - link to declaring included builds

##### Tests for `buildSrc` no longer run automatically
When Gradle builds the output of `buildSrc` it only runs the tasks that produce that output. It no longer runs the `build` task.
In particular, this means that the tests of `buildSrc` and its subprojects are not built and executed when they are not needed.

TODO - you can run these tasks from the command-line or edit buildSrc to restore the old behaviour; link to upgrade guide 

##### Init scripts are applied to `buildSrc`
```
Suggestion: update header level so these sections all live within the "improvements for buildSrc builds" seciton. Minor rewords.